src/main/java/lux/fm/bookingservice/model/Booking.java

```diff
@@ -36,4 +37,7 @@ public class Booking {
 
     @Column(name = "check_out", nullable = false)
     private LocalDate checkOut;
+
+    @Column(name = "life_time", nullable = false)
+    private LocalTime lifeTime = LocalTime.now().plusMinutes(20);
```

Comment:
Make it LocalDateTime, if we book a few minutes before 0:00 the next day we can't delete after this